test/dotcom_web/components_test.exs

```diff
@@ -30,4 +32,27 @@ defmodule DotcomWeb.ComponentsTest do
       assert html =~ ~s(<div id="#{id}" phx-hook="AlgoliaAutocomplete" phx-update="ignore">)
     end
   end
+
+  describe "error_container" do
+    test "optionally shows a title" do
+      assigns = %{
+        title: Faker.Lorem.sentence(2),
+        content: Faker.Lorem.sentence(4)
+      }
+
+      str =
+        rendered_to_string(~H"""
+        <.error_container title={assigns.title}>
+          <%= assigns.content %>
+        </.error_container>
+        """)
+
+      assert str =~ assigns.title
+      assert str =~ assigns.content
+
+      assert rendered_to_string(~H"""
```

Comment:
Nit (non-blocking): I think the assertion that would make this section justify its existence would be to somehow assert that there is no title.